server/controllers/ClientDashboardController.js

```diff
@@ -23,19 +23,28 @@ async function postGig(req, res) {
 
 async function getUsername(req, res) {
   const user = req.session.userClient;
+  console.log(user.username);
   const clientUsername = await UserClient.find({ username: user.username });
   res.json({ username: clientUsername });
 }
 
 async function getPerformerDetails(req, res) {
-  const user = req.session.userPerformer;
-  const performerInfo = await UserPerformer.find({
-    username: user.username,
-    email: user.email,
-    location: user.location,
-    about: user.about,
-  });
-  res.json({ performerInfo });
+  console.log("search id = ", req.params);
+  var id = "_.id";
+  const test = await UserPerformer.find({ _id: req.params.id });

```

Comment:
Should use findById instead of find, and since you are using await, this probably won't actually do anything until you put .exec() at the end. So it should be:
`const test = await UserPerformer.findById(req.params.id).exec();`